test/db/analysis/classes

```diff
@@ -103,7 +103,7 @@ Type Info at 0x100008370:
 type: __class_type_info
 found_at: 4295000200
 class_vtable: 4295000176
-ref_to_type_class: 13838435755002691596
+ref_to_type_class: -4608308318706860020
```

Comment:
this was and is a "random" value, isn't it? Does it have any meaning?

Reply:
I asked @thestr4ng3r who both implemented this feature and RzJson - it's apparently OK.